templates/CRM/common/l10n.js.tpl

```diff
@@ -16,7 +16,7 @@
   CRM.config.resourceBase = {$config->userFrameworkResourceURL|@json_encode};
```

Comment:
@colemanw won't these need similar handling to $lcMessages?

Reply:
Well, maybe some of them. I don't think we need to pass in `resourceBase` or `packagesBase` because if the system directories changed then assets would be getting rebuilt anyway.

Date & money preferences, yea we probably should, but it wasn't happening before which seems like an oversight, unless there was something happening to reset the cache code when those settings were changed.